block-storage-attacher/Makefile

```diff
@@ -56,7 +56,16 @@ gosec:
 
 .PHONY: buildgo
 buildgo:
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build ${GOTAGS}
```

Comment:
Remove CGO_ENABLED parameter from all go build commands when using this shared golang image